scalafmt-core/shared/src/main/scala/org/scalafmt/config/Align.scala

```diff
@@ -32,7 +32,7 @@ import metaconfig.generic.Surface
   *
   *               and want to add one extra token (for example "|>") to align by, write
   *
-  *               align.tokens."+" = [ "|> ]
+  *               align.tokens."+" = [ "|>" ]
```

Comment:
Unrelated to this work, but I just noticed the missing closing `"` when updating the Markdown config to use this syntax.

Reply:
Removed from this changeset.